pkg/metrics/metrics.jsx

```diff
@@ -238,6 +238,18 @@ function make_rows(rows, rowProps, columnLabels) {
     );
 }
 
+async function get_pcp_packages() {
+    const os_release = await read_os_release();
+    const pcp_packages = ["pcp"];
+
+    // PCP contains the Python module on Arch Linux, for all other distro's it is split up.
```

Comment:
For our future selves: Please document why we have to explicitly install both: That on Fedora, python3-pcp depends on pcp, and on Debian pcp depends on python3-pcp.